includes/widgets/playing-cards.php

```diff
@@ -0,0 +1,710 @@
+				'type' => \Elementor\Controls_Manager::SLIDER,
+				'size_units' => [ 'px', 'em', 'rem', 'custom' ],
+				'range' => [
+					'px' => [
+						'min' => 0,
+						'max' => 50,
+						'step' => 1,
+					],
+					'em' => [
+						'min' => 0,
+						'max' => 100,
+					],
+					'rem' => [
+						'min' => 0,
+						'max' => 100,
+					],
+				],
+				'selectors' => [
+					'{{WRAPPER}} .e-playing-cards-container' => '--e-card-suit-font-size: {{SIZE}}{{UNIT}};',
+				],
+			]
+		);
+		$this->add_control(
+			'suit_divider',
+			[
+				'type' => \Elementor\Controls_Manager::DIVIDER,
+			]
+		);
+		$this->add_responsive_control(
+			'diamond_color',
+			[
+				'label' => esc_html__( 'Diamond ', 'elementor' ) . static::$suit_char_map['diamond'],
+				'type' => \Elementor\Controls_Manager::COLOR,
+				'selectors' => [
+					'{{WRAPPER}} .e-playing-cards-container' => '--e-card-diamond-color: {{VALUE}}',
+				],
+				'default' => '#ff0000',
+			]
+		);$this->add_responsive_control(
+			'heart_color',
+			[
+				'label' => esc_html__( 'Heart ', 'elementor' ) . static::$suit_char_map['heart'],
+				'type' => \Elementor\Controls_Manager::COLOR,
+				'selectors' => [
+					'{{WRAPPER}} .e-playing-cards-container' => '--e-card-heart-color: {{VALUE}}',
+				],
+				'default' => '#ff0000',
+			]
+		);$this->add_responsive_control(
+			'club_color',
+			[
+				'label' => esc_html__( 'Club ', 'elementor' ) . static::$suit_char_map['club'],
+				'type' => \Elementor\Controls_Manager::COLOR,
+				'selectors' => [
+					'{{WRAPPER}} .e-playing-cards-container' => '--e-card-club-color: {{VALUE}}',
+				],
+				'default' => '#000000',
+			]
+		);$this->add_responsive_control(
+			'spade_color',
+			[
+				'label' => esc_html__( 'Spade ', 'elementor' ) . static::$suit_char_map['spade'],
+				'type' => \Elementor\Controls_Manager::COLOR,
+				'selectors' => [
+					'{{WRAPPER}} .e-playing-cards-container' => '--e-card-spade-color: {{VALUE}}',
+				],
+				'default' => '#000000',
+			]
+		);
+		$this->end_controls_section();
+		$this->start_controls_section(
+			'royal_suit_offset',
+			[
+				'label' => esc_html__( 'Royal Suit Offset', 'elementor' ),
+				'tab' => \Elementor\Controls_Manager::TAB_STYLE,
+			]
+		);
+		$this->add_responsive_control(
+			'royal_offset_h',
+			[
+				'label' => esc_html__( 'Horizontal Offset', 'elementor' ),
+				'type' => \Elementor\Controls_Manager::SLIDER,
+				'size_units' => [ 'px', 'em', 'rem', '%', 'custom' ],
+				'range' => [
+					'px' => [
+						'min' => 0,
+						'max' => 50,
+						'step' => 1,
+					],
+					'em' => [
+						'min' => 0,
+						'max' => 100,
+					],
+					'rem' => [
+						'min' => 0,
+						'max' => 100,
+					],
+					'%' => [
+						'min' => 0,
+						'max' => 100,
+					],
+				],
+				'selectors' => [
+					'{{WRAPPER}} .e-playing-cards-container' => '--e-card-royal-suit-offset-h: {{SIZE}}{{UNIT}};',
+				],
+			]
+		);
+		$this->add_responsive_control(
```

Comment:
You can add an empty line between functions, which makes the code more readable.